app/src/main/java/com/medmapper/v33001/MainViewModel.kt

```diff
@@ -18,7 +18,7 @@ class MainViewModel(var medicineService : IMedicineService) /*= MedicineService(
 
     internal val NEW_MEDICATION = "New Medication"
     var medicine : MutableLiveData<List<Medicine>> = MutableLiveData<List<Medicine>>()
-    var selectedMedicine by mutableStateOf(Medicine())
+    private var selectedMedicine by mutableStateOf(Medicine())
```

Comment:
Making the mutableStateOf variable private makes sure changes to it are in one class only. 

[(https://developer.android.com/kotlin/coroutines/coroutines-best-practices)](https://developer.android.com/kotlin/coroutines/coroutines-best-practices#mutable-types)

Reply:
A mutableState object will likely be accessed by an Activity class, as that's the nature of having a mutableState in the first place.  private will hide this access.  I urge caution when changing an access modifier in a ViewModel.